src/styles/core/_vars.scss

```diff
@@ -72,6 +72,7 @@ $nav-card-vertical-spacing: 10px;
 $nav-card-horizontal-spacing: 10px;
 $nav-card-horizontal-spacing-large: $nav-card-horizontal-spacing * 2;
 $nav-card-horizontal-spacing-small: $nav-card-horizontal-spacing * 0.5;
+$nav-time-transition: 0.15s;
```

Comment:
Do we need a separate SASS variable for this or can we just consolidate to the CSS property defined in `DocumentationLayout` directly?

Reply:
True!